### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by saketkum11**
wishlist 
- add item
- remove item

### CODE REVIEW

**Review by shraddha15-creator — APPROVED** [review]
code is clean and understandable 
used good variable names

File: src/Context/Auth/Auth.js [hunk 1]
```diff
@@ -29,7 +39,7 @@ const fetchLoginDetails = async (email,password) => {
     // saving the encodedToken in the localStorage
     
     localStorage.setItem("token", response.data.encodedToken);
-    localStorage.getItem("token", response.data.encodedToken);
+   // setAuthToken(localStorage.getItem("token"))
      navigate("/");
```
Comment (JowelTisso) [inline src/Context/Auth/Auth.js]: You can extract localStorage as separate function

**Review by JowelTisso — APPROVED** [review]
Rest Looks Good

File: src/Context/Auth/Auth.js (hunk 1 above)
Comment (saketkum11) [reply src/Context/Auth/Auth.js]: okk sure

**Pull request merged**